nikola/nikola.py

```diff
@@ -656,6 +657,7 @@ def __init__(self, **config):
 
         self.TRANSLATABLE_SETTINGS = ('BLOG_AUTHOR',
                                       'BLOG_TITLE',
+                                      'BLOG_EMAIL',
```

Comment:
This one and `_GLOBAL_CONTEXT_TRANSLATABLE` (line 696) need to be removed as well.

Reply:
oh, missed this one.